src/appengine/libs/csp.py

```diff
@@ -77,9 +77,13 @@ def get_default_builder():
   builder.add('img-src', 'self', quote=True)
   builder.add('manifest-src', 'self', quote=True)
 
-  # External scripts. Google analytics and charting libraries we depend on.
+  # External scripts. Google analytics, charting libraries, and code formatter.
   builder.add('script-src', 'www.google-analytics.com')
   builder.add('script-src', 'www.gstatic.com')
+  builder.add('script-src', 'cdn.rawgit.com')
```

Comment:
This probably ought to have a TODO comment. AFAICT, it's a complete CSP bypass, since anyone can host any code they want on there.

Reply:
Good catch! I'll move this to the group of unsafe things we need to fix.